deploy-board/deploy_board/webapp/common.py

```diff
@@ -135,6 +135,17 @@ def restart(request, name, stage):
 def rollback_to(request, name, stage, deploy_id):
     query_dict = request.POST
     desc = query_dict.get('description', None)
+    mark_build_as_bad = query_dict.get('mark_build_as_bad', True)
```

Comment:
suggest mark_build_as_bad = true if query_dict.get('mark_build_as_bad', 'on') == 'on else False and removing the  if statements below. It is not ideal to mix the type 'off' and True on the same variable.